delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/TestHelpers.scala

```diff
@@ -33,21 +25,6 @@ trait TestHelpers {
     inner("", length)
   }
 
-  /**
-    * Loads the content of the argument classpath resource as a string.
-    *
-    * @param resourcePath the path of a resource available on the classpath
-    * @return the content of the referenced resource as a string
-    */
-  final def contentOf(resourcePath: String): String = {
```

Comment:
If the `ClasspathResourceUtils` would return a `Task[A]`, I would still retain the current signatures and call `ClasspathResourceUtils.contentOf(...).hideErrors.unsafeRunSync()`